kernel/classes/ezrssexport.php

```diff
@@ -449,6 +449,10 @@ function generateFeed( $type )
                 $creatorObject->attribute('name'), ENT_NOQUOTES, 'UTF-8'
             );
         }
+        else
+        {
+            $author->name = "Admin from " . $config->variable( 'SiteSettings', 'SiteName' );
```

Comment:
Avoiding from crashing is great :)
I have no preference regarding the fallback message as long as we can customize it a bit. We are dealing with an edge case as user should not be deleted in the first place.